### DISCUSSION TITLE
Number Conversion extension

### DISCUSSION THREAD

**Discussion by PURPL3-CAT** [TurboWarp/extensions#1624] [Show and tell]
I’ve created a new ‘Number Conversion’ extension. It’s designed to handle large numbers, making it great for scoreboards or currency in games like clicker games where displaying a lot of digits is not an option. The extension can convert numbers into short form, comma-separated form, and word form, and it **works both ways**. I’ve added it to my GitHub, feel free to check it out and let me know what you think!

Link to code: [github.com/PURPL3-CAT/Number-Conversion/](https://github.com/PURPL3-CAT/Number-Conversion/)

![Screenshot 2024-07-23 174545](https://github.com/user-attachments/assets/7ef4e228-6dff-4501-94ac-5fa654a5c217)

**Comment by Jax3dJaxHahaPlanetBall** [comment]
Woah! Yippe!
